docker-compose.yml

```
@@ -2,30 +2,30 @@ version: '3.4'
 services:
   db:
     environment:
-      - MYSQL_USER=foreman
-      - MYSQL_PASSWORD=foreman
-      - MYSQL_DATABASE=foreman
-      - MYSQL_DATADIR_ACTION=upgrade-auto
+      - POSTGRES_USER=foreman
+      - POSTGRES_PASSWORD=foreman
+      - POSTGRES_DATABASE=foreman
+      - PGDATA=/var/lib/postgresql/data/pgdata
     hostname: db.example.com
-    image: centos/mariadb-102-centos7
+    image: postgres:10
     ports:
-      - 3306
+      - 5432
     restart: always
     healthcheck:
       test: ["CMD-SHELL", "nc -z 127.0.0.1 3306 || exit 1"]
```

Postgres is port 5432
```suggestion
      test: ["CMD-SHELL", "nc -z 127.0.0.1 5432 || exit 1"]
```

---

thanks, updated.